src/import/cn-cbor/src/cn-get.c

```diff
@@ -20,6 +20,7 @@ cn_cbor* cn_cbor_mapget_int(const cn_cbor* cb, int key) {
       if (cp->v.uint == (unsigned long)key) {
         return cp->next;
       }
+      break;
     case CN_CBOR_INT:
```

Comment:
this should be ok. BTW. this is a third party tool. Will be hard to track these changes. Also look at that https://github.com/cabo/cn-cbor/blob/master/src/cn-get.c they've already have this brake at place.

Reply:
Wow, Oct 11 2017: https://github.com/cabo/cn-cbor/commit/2e513d9d7a02e4ae74afa638d02d2b724b4fee92#diff-997e37b9907c2edf99c5eca675426940

Reply:
Hm, this means an update in the cn-cbor lib would solve this issue. And could introduce further ones. Let's keep this solution for now 👍

Reply:
Hah! nice!